Formula/postgresql@9.4.rb

```diff
@@ -29,11 +29,11 @@ class PostgresqlAT94 < Formula
   uses_from_macos "perl"
 
   on_linux do
+    depends_on "autoconf@2.69" => :build
```

Comment:
Can we add a comment that this is only needed for the `configure` patch? It can be in a follow-up.

Reply:
I figured a comment would be unnecessary because these versions of postgresql are already deprecated, so the patches won't go away (probably). Up to you though, just let me know.

There's also the option of re-generating `configure` manually and making a patch from that, but that's more work than I'm willing to put in for a deprecated package lol